app/vmctl/README.md

```diff
@@ -508,6 +512,21 @@ Instead, use [relabeling in VictoriaMetrics](https://github.com/VictoriaMetrics/
 4. When importing in or from cluster version remember to use correct [URL format](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#url-format)
 and specify `accountID` param.
 
+## Verifying exported blocks from VictoriaMetrics
+
+ VictoriaMetrics allows exporting data block in native format [link](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-export-data-in-native-format). 
+You can verify exported data at disk before uploading it by `vmctl` `-verify-blocks` command:
+
+```bash
+# export blocks from VictoriaMetrics
+curl localhost:8428/api/v1/export/native -g -d 'match[]={__name__!=""}' -o data_block
+# verify block content
+./vmctl verify-blocks --blockPath data_block
```

Comment:
can we use args (for example, c.Args().First()) as path instead of specifying a flag?
See:
```
Action: func(c *cli.Context) error {
					common.StartUnmarshalWorkers()
					blockPaths := c.Args().First()
```

Reply:
Done